### DISCUSSION TITLE
Intro - cblanken

### DISCUSSION THREAD

**Discussion by cblanken** [Tatoeba/tatoeba2#3111] [General]
Hello :wave:, I'm cblanken. I'm from the United States. Unfortunately I only speak English at the moment, but I'm currently learning German and hope to revive some of the Spanish I learned way back in high school at some point.

I found Tatoeba a while ago. I think it was via [Clozemaster](https://www.clozemaster.com) since it uses the Tatoeba corpus. Since then I've been using Tatoeba while learning German and it has been quite helpful, so I thought I could contribute.

I haven't done any web dev professionally, but I am a software developer, and I've done some side projects with vanilla JS/CSS and dabbled with React a bit. I've also worked with a couple Python web frameworks, Flask and Django.

Anyway, I'd like to help out. I've setup the TatoVM per the contributor's guide and am planning to familiarize myself with CakePHP and the codebase in general over the next several weeks. If there are any good beginner issues I can take a look at, please let me know. Otherwise I'll just muck around until I find something interesting to work on :)

**Comment by trang** [comment]
Hello @cblanken 👋 and welcome!  

Thanks for offering your help!

For your first issue, I can suggest one of these:
* https://github.com/Tatoeba/tatoeba2/issues/1726 - This one was an issue attempted by another contributor before, so you'll find the solution in the comments. But the contributor never opened a pull request. The main point of tackling this issue is really just so that you get familiar with the pull request process.
* https://github.com/Tatoeba/tatoeba2/issues/2275 - With this one you will learn a bit more about how CakePHP works. It involves retrieving some info about audio recordings from the database and displaying the info on the sentence's page.
* The issue that you've come across with uploading an avatar image (https://github.com/Tatoeba/tatoeba2/issues/3100) could also be a good one to work on as your first issue.

We otherwise have a list of beginner issues listed under the label "[good first issue](https://github.com/Tatoeba/tatoeba2/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22)". You can have a look and see if anything else catches your interest 🙂

**Reply by cblanken** [reply]
Thanks trang!
I'll look into one of these and see how it goes.
